index.html

```diff
@@ -13,7 +13,24 @@
         <link href="css/theme-red.css" rel="stylesheet" type="text/css" media="all" />
         <link href="css/custom.css" rel="stylesheet" type="text/css" media="all" />
         <link href='http://fonts.googleapis.com/css?family=Lato:300,400%7CRaleway:100,400,300,500,600,700%7COpen+Sans:400,500,600' rel='stylesheet' type='text/css'>
-    </head>
+    <style>
+	.video-container {
+    position: relative;
```

Comment:
Indentation out of place.

Reply:
Also you should put all the custom css styles that you are implementing in the css/custom.css file.